pkg/server/container_attach.go

```diff
@@ -62,7 +62,7 @@ func (c *criContainerdService) attachContainer(ctx context.Context, id string, s
 	}
 	handleResizing(resize, func(size remotecommand.TerminalSize) {
 		if err := task.Resize(ctx, uint32(size.Width), uint32(size.Height)); err != nil {
-			glog.Errorf("Failed to resize task %q console: %v", id, err)
+			logrus.WithError(err).Errorf("Failed to resize task %q console", id)
```

Comment:
Why not use log.G(ctx)?

Reply:
Because the log context from containerd is not passed to cri-containerd yet.
It seems that we need to integrate with the [`interceptor`](https://github.com/containerd/containerd/blob/master/server/server.go#L238).